app/containers/HomeContainer/index.js

```diff
@@ -74,6 +73,22 @@ export function HomeContainer({
   };
   const debouncedHandleOnChange = _.debounce(handleOnChange, 200);
 
+  const activateLasers = () => {
+    // console.log('hello');
+    return (
+      <Search
+        data-testid="search-bar"
+        defaultValue={repoName}
+        type="text"
+        onSearch={searchText => {
+          // console.log('!313232'+ searchText);
```

Comment:
remove this

Reply:
there is a shorthand for such stuff. 


what you can do is use something like this 
```
onSearch={debouncedHandleOnChange}
```
it means that whatever the variables you get in the onSearch callback should be passed as is to `debouncedHandleOnChange`

Reply:
I remove this.